### DISCUSSION TITLE
NextcloudAIO - ClamAV errors in logs

### DISCUSSION THREAD

**Discussion by Aceriz** [nextcloud/all-in-one#4828] [Questions] [answered]
I have my setup running on unraid.  

I have noticed and have been getting thousands of errors in the logs that I think are all related to the CLAMav antivirus 
I get about 4 x per hour the following error sets...  I am not sure how to assess and rectify this

[PHP] Error: opendir(/mnt/user/appdata/clamav/tmp.9097211a46): Failed to open directory: Permission denied at /var/www/html/lib/private/Files/Storage/Local.php#164

[files] Warning: User NextcloudAIO still has unscanned files after running background scan, background scan might be stopped prematurely
	

Any help or way to work through the diagnostic of this would be greatly appreciated

Thanks

**Comment by Aceriz** [comment]
I think but am not sure if this is related to the CLAMav within nextcloudaio trying to access files that are on my external files location...  Where on my main server I have a CLAMav docker running

**Comment by Aceriz** [comment]
if this is the case is there a way to blacklist this folder location? within the nextcloudaio clamav

**Comment by szaimen** [comment, marked as answer]
Hi, please ask this at https://help.nextcloud.com thanks!
